internal/policy/policy.go

```diff
@@ -855,6 +853,61 @@ func (s *State) getRootVerifier() (*Verifier, error) {
 	}, nil
 }
 
+func Apply(ctx context.Context, repo *git.Repository, signRSLEntry bool) error {
+	_, err := LoadCurrentState(ctx, repo, PolicyStagingRef)
```

Comment:
I was using LoadCurrentState since it also verifies the state of the repo, which means that if this fails, apply should not be run, since verification failed on the policy staging ref

Reply:
I see. That's a good call, let's keep this thread open, especially in the context of #308.